README.md

```diff
@@ -681,6 +682,10 @@ Form component supports the following html attributes:
   schema={} />
 ```
 
+### Enum fields
+
+String fields that use `enum` and a `select` widget will have an empty option in the options list. When a user selects that option, the field will be set to `undefined` (similar to how regular `string` fields work if the field is empty). This also means that if you have an empty string in your `enum` array, selecting that option will cause the field to be set to `undefined`.
```

Comment:
I think I'd move this in new section under the *Form data validation*, eg. *The case of empty strings*.